.github/pull_request_template.md

```diff
@@ -1,3 +1,35 @@
 <!-- 
 Thanks for your contribution!
+Please fill out the neccessary sections, and delete unused ones.
 -->
+Fixes #issue-number
+----------
+## Type of change:
+- [ ] Bug fix (non-breaking change which fixes an issue).
+- [ ] New feature (non-breaking change which adds functionality).
```

Comment:
There is not concept of breaking change in saskatoon right now. So I don;t think we should add this stuff in perenthesis.

Reply:
Wouldn't for example dependency changes or some major changes break the environment for some people?
I could maybe add this instead:

```markdown
- [ ] Change to the app models (requires making migrations).
```
What do you think? @tristanlatr

Reply:
Yes that sounds good